src/backend/xilinx/xml.rs

```diff
@@ -150,7 +150,7 @@ impl Backend for XilinxXmlBackend {
                 name: axi_name,
                 mode: "master",
                 range: "0xFFFFFFFFFFFFFFFF",
-                data_width: 64,
+                data_width: 512, //matches actual hardware width
```

Comment:
Can we leave a comment here explaining why this needs to be 512? Ideally pointing to documentation somewhere?

Reply:
Would this be good? Is there a deeper reason "why" that should be addressed?
```suggestion
                // Width should match the bus data width of memory modules
                // described in hardware, for example see
                // https://github.com/cucapra/calyx/blob/c2b12a0fe6b1ee3aaaae0c66e7c4619ee6c82614/src/backend/xilinx/toplevel.rs#L58
                data_width: 512,
```

Reply:
That works!